### ISSUE TITLE
placement?

### ISSUE THREAD

**Issue by kthffmn** [learn-co-curriculum/myQuery#3]
We're having students build out what jQuery accomplishes before we show them what jQuery does. I know this format is standard for the Ruby section (for instance, we have them make a simplified version of ActiveRecord) but we usually build up to it. 

Since we're not having them interact with the DOM until we throw them into jQuery, perhaps we should make this the last lab in the JavaScript > jQuery unit, not the first.

**Comment by BenjaGross** [comment]
@blake41 @MsUzoAgu I agree with @kthffmn for this. It seems way better to do this lab after introducing jQuery. We did it that way for web-0615 and it worked out pretty well.

**Comment by AnnJohn** [comment]
Iceboxing for now
